scalyr_agent/copying_manager/copying_manager.py

```diff
@@ -388,10 +388,20 @@ def set(self, path, worker_id, value):
         self.__paths[path][worker_id] = value
 
     def get_path(self, path):
+
+        # We don't want to be creating an empty default dict by get.
+        if path not in self.__paths:
+            return {}.items()
+
         return self.__paths[path].items()
```

Comment:
Up to you but I think this is ok: `return self.__paths.get(path, {}).items()`

Reply:
Yes, definitely!